Playground/index-local.html

```diff
@@ -55,6 +55,12 @@
         <div id="host-element">
         </div>
         <script>
+            function GetAbsoluteUrl(url) {
+                const a = document.createElement("a");
+                a.href = url;
+                return a.href;
+            }
+

```

Comment:
I know this is copied this from elsewhere, but I just noticed that this won't work in Babylon Native. We will need to do something different here and elsewhere.